target/linux/pistachio/base-files/lib/functions/mac.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
This was refactored out for duplicate code in 6lowpan right? Therefore please add the refactoring for that too.